.changeset/fuzzy-crabs-divide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Added some varients for content on the right side of the Hero Components. This change adds css and the template updates to render the hero samples on component and pattern pages for heros.
```

Comment:
Once we have the right name, can we condense this to something more simple? 

The changes to the documentation should have its own changeset.